dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/validation/hooks/PreCheckSecurityOwnershipValidationHook.java

```diff
@@ -461,9 +463,14 @@ private void checkEventWriteAccess( ValidationErrorReporter reporter, Event even
         checkNotNull( user, USER_CANT_BE_NULL );
         checkNotNull( programStage, PROGRAM_STAGE_CANT_BE_NULL );
         checkNotNull( programStage.getProgram(), PROGRAM_CANT_BE_NULL );
-        checkNotNull( eventOrgUnit, ORGANISATION_UNIT_CANT_BE_NULL );
 
-        if ( isCreatableInSearchScope ? !organisationUnitService.isInUserSearchHierarchyCached( user, eventOrgUnit )
+        if ( eventOrgUnit == null )
```

Comment:
If this is a temporary fix or a possible temporary fix, it would be better to add a TODO to remember that we need to have another look

Reply:
we can add it but the test added should fail when we'll update that part later on

Reply:
I was just saying that we should have something to remind us that this is temporary but I just saw that https://jira.dhis2.org/browse/DHIS2-12600 was already created, so no need for a TODO

Reply:
yes we have Jira and acceptance criteria so let's just use those instead of TODO that most of the time is forgetten